Services/AccessControl/classes/class.ilObjRoleTemplate.php

```diff
@@ -46,6 +46,15 @@ public function __construct($a_id = 0, $a_call_by_reference = false)
     }
 
 
+    /**
+     * return translated title for autogenerated templates
+     * @return string
```

Comment:
Please remove the phpdoc-comment (\return string), since it does not provide additional information.